uxarray/core/api.py

```diff
@@ -14,10 +14,7 @@
 
 def open_grid(grid_filename_or_obj: Union[str, Path, xr.DataArray, np.ndarray,
                                           list, tuple],
-              gridspec: Optional[str] = None,
-              vertices: Optional[list] = None,
-              islatlon: Optional[bool] = False,
-              isconcave: Optional[bool] = False,
+              latlon: Optional[bool] = False,
```

Comment:
This newer name looks better; however, similar to another comment in grid.oy, we have already had these self.islatlon in the user API. Renaming it (even though it is a small thing) is actually not backwards-compatible (we may break the existing user workflows with such simple var name changes). Having a DeprecationWarning for a few version releases into the future before such changes is always a good practice.

Reply:
> Having a DeprecationWarning for a few version releases into the future before such changes is always a good practice.

As per our discussion the other day, I'll look into this and add it in.